Sources/Subprocess/AsyncBufferSequence.swift

```diff
@@ -10,22 +10,21 @@
 //===----------------------------------------------------------------------===//
 
 #if canImport(System)
-import System
+@preconcurrency import System
 #else
 @preconcurrency import SystemPackage
 #endif
 
 #if SubprocessSpan
 @available(SubprocessSpan, *)
 #endif
-internal struct AsyncBufferSequence: AsyncSequence, Sendable {
-    internal typealias Failure = any Swift.Error
-
-    internal typealias Element = SequenceOutput.Buffer
+public struct AsyncBufferSequence: AsyncSequence, Sendable {
```

Comment:
I wonder how we can remind ourselves better why this is now an API since it's not covered in the proposal

Reply:
I will start the CHANGELOG file to track API changes.